Graphs/VariationTextGraph/index.js

```diff
@@ -109,7 +106,7 @@ export class VariationTextGraph extends AbstractGraph {
     }
 
     numberWithCommas(x) {
-        return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
+        return x && x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
```

Comment:
I think we need to return undefined if `!x`

Reply:
@natabal 

If !x, then it will display NaN on the graph.